src/java/org/apache/cassandra/index/sai/disk/v1/bbtree/BlockBalancedTreeWriter.java

```diff
@@ -704,24 +709,24 @@ private int runLen(int start, int end, int byteOffset)
 
         // The following 3 methods are only used when DEBUG is true:
 
-        private void valueInBounds(byte[] packedValue, int packedValueOffset, byte[] minPackedValue, byte[] maxPackedValue)
+        private void valueInBounds(byte[] packedValues, int packedValueOffset, byte[] minPackedValue, byte[] maxPackedValue)
```

Comment:
Here and in all these cases below, these are the bytes of one packed value, right? If that's the case, `packedValues` worries me a little?

Reply:
No, these are the bytes of an array of packed values and use the `packedValueOffset` to index to the correct packed value.